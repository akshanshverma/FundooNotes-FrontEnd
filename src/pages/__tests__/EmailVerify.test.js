import React from 'react';
import { shallow } from 'enzyme';
import EmailVerify from "../EmailVerify";


const component = shallow(<EmailVerify debug />);
describe('EmailVerify', () => {
    it('should render correctly in "debug" mode emailVerify', () => {
        expect(component).toMatchSnapshot();
    });
});