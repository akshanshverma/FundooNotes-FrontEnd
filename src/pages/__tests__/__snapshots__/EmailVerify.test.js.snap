// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`EmailVerify should render correctly in "debug" mode emailVerify 1`] = `
<div
  className="maindiv"
>
  <WithStyles(Card)
    id="card"
  >
    <div>
      <WithStyles(Typography)
        color="primary"
        id="loginT"
      >
        Account Verification
      </WithStyles(Typography)>
      <div
        className="hold"
      >
        <div
          className="button"
        >
          <WithStyles(Button)
            color="primary"
            onClick={[Function]}
            type="Submit"
            variant="contained"
          >
            Activate Account
          </WithStyles(Button)>
        </div>
        <div
          className="errMsg"
        />
      </div>
    </div>
  </WithStyles(Card)>
</div>
`;
